### ISSUE TITLE
output is 2 .js files with 'undefined' as the content

### ISSUE THREAD

**Issue by joej** [bguiz/modernizr-cli#4]
Mac OS X 10.11.4
TL;DR; - output files contain the word 'undefined' and no js

Details:
I tried 2 ways -- 1 was via the web page builder and copying the 'Command Line Config'
(note: this prepends 'test/' in front of all the test options) and the other was as the README suggested as ...
- npm install -g modernizr-cli
- cp /usr/local/lib/node_modules/modernizr-cli/node_modules/modernizr/lib/config-all.json modernizr-config-all.json
- /usr/local/bin/modernizr modernizr-config-all.json A B

output is:
CLI arguments: [ 'modernizr-config-all.json', 'A', 'B' ]
File names: { config: 'modernizr-config-all.json',
  output: 'A',
  minifiedOutput: 'B' }

Content of files A and B are both the simple word 'undefined' with no trailing carriage return.
